src/Status/Views.elm

```diff
@@ -0,0 +1,39 @@
+module Status.Views exposing (view)
+
+import Html exposing (Html, text, button, div, li, ul, b)
+import Html.Events exposing (onClick)
+import Status.Types exposing (StatusMsg(GetStatus), StatusResponse)
+import Types exposing (Msg(MsgForStatus), Model)
+
+
+view : Model -> Html Types.Msg
+view model =
+    div []
+        [ button [ onClick (MsgForStatus <| GetStatus) ] [ text "Get Status" ]
```

Comment:
My initial thought about this mapping from `StatusMsg` to `MsgForStatus StatusMsg` was that we can just return `Html StatusMsg`, but then we aren't able to return any top level `Msg`, which is the problem the translator pattern is trying to solve. But that pattern itself requires every message to be tagged with `ForParent` or `ForSelf`.

This is a long way of saying, I think this pattern has the same effect, the same implementation in code (mapping `StatusMsg`s), but skips having translators.

This seems like a positive to me. I still need to read that blog post you sent me, but this is immediately more understandable and similar to how I first wanted to separate out the components (but had no concept of how to do it with the type system :))